tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -253,6 +264,21 @@ private void CheckUnboundProbes()
             }
         }
 
+        private void AcceptConfiguration(ProductConfigChangedEventArgs args)
```

Comment:
This code is very RCM specific - and doesn't seem like it would be different  in any of the other products. Can we move out it out `LiveDebugger` so that we only deal with `ProbeConfiguration`s directly in this class?

Reply:
Refactored at 089c7559d5535aac3c5c45a113b32dd11f9b7837, please review

Reply:
Great refactoring! Love it.